integrations/openai/src/index.ts

```diff
@@ -1,17 +1,22 @@
 import { llm } from '@botpress/common'
 import { interfaces } from '@botpress/sdk'
 import OpenAI from 'openai'
-import { ModelId } from './schemas'
+import { LanguageModelId, ImageModelId } from './schemas'
 import * as bp from '.botpress'
+import { InvalidPayloadError } from '@botpress/client'
+import { ImageGenerateParams, Images } from 'openai/resources'
 
 const openAIClient = new OpenAI({
   apiKey: bp.secrets.OPENAI_API_KEY,
 })
 
+const DefaultLanguageModelId: LanguageModelId = 'gpt-4o-mini-2024-07-18'
```

Comment:
**nitpick:** I don't like using pascal case for variables and constants. I prefer camelCase for variables and/or SCREAMING_SNAKE_CASE for constants (pick one)

Reply:
I think we've started using screaming case in skynet, so I'll use that one.

Reply:
Done in https://github.com/botpress/botpress/pull/13200/commits/4818294e6ce5315bfb3b97debb38bb794808ced5